support-workers/src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -112,61 +115,64 @@ object DigitalSubscriptionEmailAttributes {
 
 }
 class DigitalPackEmailFields(
```

Comment:
this file hasn't really changed, it's just shuffled a bit.  (In another PR) it should probably have a separate file for each type of digital subscription execution rather than such a big class.